src/pranitaPJun20/MeetingRoomOverloading.java

```diff
@@ -0,0 +1,67 @@
+package pranitaPJun20;
+
+public class MeetingRoomOverloading {
+	int bookingTime;
+	String selectedMeetingRoom;
+	String[] rooms = { "Nalanda", "Takshashila", "AgraFort", "PratapGadh" };
+
+	void bookMeetingRoom() {
+		selectedMeetingRoom = "Nalanda";
+		bookingTime = 1;
+		displayInfo();
+		for (int index = 0; index < rooms.length; index++) {
+			if (rooms[index].equals("Nalanda")) {
+				rooms[index] = " ";
+			}
+		}
+	}
+
+	void bookMeetingRoom(String meetingRoomName) {
+		if (!meetingRoomName.equals("Nalanda")) {
+			selectedMeetingRoom = meetingRoomName;
+			bookingTime = 1;
+			for (int index = 0; index < rooms.length; index++) {
+				if (rooms[index].equals(meetingRoomName)) {
+					rooms[index] = " ";
+				}
+			}
+			displayInfo();
+		} else {
+			System.out.println("\nThe meeting room Nalanda is already booked.Please book any other meeting room");
+		}
+
+	}
+
+	void bookMeetingRoom(String meetingRoomName, int time) {
+		selectedMeetingRoom = meetingRoomName;
+		bookingTime = time;
+		for (int index = 0; index < rooms.length; index++) {
+			if (rooms[index].equals(meetingRoomName)) {
+				rooms[index] = " ";
```

Comment:
Hi @PranitaPJun20 ,
Here once you entered in for loop **break** it because no need to iterate for other rooms,

Reply:
Pushed new file with proper changes.